lib/mlir/Dialect/Clift/Utils/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+add_mlir_dialect_library(
```

Comment:
Is it necessary to use this? I guess we need this only if we use MLIR tablegen magic.

Let's use our `revng_add_library` function wherever we can.

Reply:
Done.